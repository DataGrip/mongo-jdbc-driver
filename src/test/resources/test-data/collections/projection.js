// before
db.col.insertOne({key: "value", array: [1, 2, 3, {another_object: "  .$# "}]});
// command
db.col.find({}, {_id: 0});
// clear
db.col.drop();