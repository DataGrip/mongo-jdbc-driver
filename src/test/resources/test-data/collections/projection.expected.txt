|map|
|Document{{key=value, array=[1, 2, 3, Document{{another_object=  .$# }}]}}|